jsonb-platform-tck/pom.xml

```
@@ -0,0 +1,375 @@
+                    <skip>true</skip>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <configuration>
+                    <systemPropertyVariables>
+                        <payara.home>${payara.home}</payara.home>
+                        <junit.log.traceflag>true</junit.log.traceflag>
+                        <harness.log.traceflag>true</harness.log.traceflag>
+                        <cts.harness.debug>true</cts.harness.debug>
+                    </systemPropertyVariables>
+                </configuration>
+                <executions>
+
+                    <execution>
+                        <id>cdi-tests-javatest</id>
+                        <goals>
+                            <goal>integration-test</goal>
+                            <goal>verify</goal>
+                        </goals>
+                        <configuration>
+                            <dependenciesToScan>jakarta.tck:jsonb-platform-tck</dependenciesToScan>
+                            <groups>${javatest-testGroups}</groups>
+                            <includes>
+                                <include>com/sun/ts/tests/jsonb/cdi/**/*Servlet*.java</include>
+                                <include>com/sun/ts/tests/jsonb/cdi/**/*Jsp*.java</include>
+                            </includes>
+                        </configuration>
+                    </execution>
+
+                    <execution>
+                        <id>pluggability-tests-javatest</id>
+                        <goals>
+                            <goal>integration-test</goal>
+                            <goal>verify</goal>
+                        </goals>
+                        <configuration>
+                            <dependenciesToScan>jakarta.tck:jsonb-platform-tck</dependenciesToScan>
+                            <groups>tck-javatest</groups>
+                            <includes>
+                                <include>com/sun/ts/tests/jsonb/pluggability/jsonbprovidertests/*Servlet*.java</include>
+                                <include>com/sun/ts/tests/jsonb/pluggability/jsonbprovidertests/*Jsp*.java</include>
+                            </includes>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>web</id>
+            <properties>
+                <javatest-testGroups>web</javatest-testGroups>
+            </properties>
+        </profile>
+        <profile>
+            <id>appclient</id>
+
+            <dependencies>
+                <dependency>
+                    <groupId>jakarta.tck.arquillian</groupId>
+                    <artifactId>arquillian-protocol-appclient</artifactId>
+                    <version>${jakarta.tck.arquillian.version}</version>
+                </dependency>
```

Missing a dependency for the jsonp-platform-tck below
```suggestion
                </dependency>
                <dependency>
                    <groupId>jakarta.tck</groupId>
                    <artifactId>jsonp-platform-tck</artifactId>
                    <version>${jakarta.tck.platform.version}</version>
                </dependency>
```